### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by marykatefain**
Total redesign of website. Includes new Services and About pages, new imagery and language, and new color themes. It's also now using nunjucks for templating, and things have been refactored to be more "atomic" as the site expands (though not entirely, just to a degree).

It could possibly still use some "finishing" so @alexgleason would like to take a look with your eye for design you can probably spot some areas of improvement. However I think it's good to merge and deploy now and then we can continue working from master. The mobile styles could also use some improving, but once again I think it will work for now.

Just a couple screenshots for reference:

screenshot from 2017-12-15 17 34 10

screenshot from 2017-12-15 17 33 57

### CODE REVIEW

File: www/about.njk [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            <h3>Worker's Collective</h3>
+          </div>
+          <p>At Candlewaster, the workers are the owners. No corporate evil to see here.</p>
+        </div>
+      </div>
+
+    </div>
+  </div>
+
+<div class="wide-image quote" id="lennon">
```
Comment (alexgleason) [inline www/about.njk]: needs indentation

File: www/assets/styles/card_styles.scss [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+@media (max-width: 600px) {
+  .services.card {
+    width: auto;
+    height: auto;
+    margin-bottom: 20px;
+  }
+}
+
+
+// .portfolio.card {
```
Comment (alexgleason) [inline www/assets/styles/card_styles.scss]: wanna just remove this? it's always in the git history, but inactive code without an explanation will be unhelpful 4 months from now 😛

File: www/assets/styles/card_styles.scss [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+//   .card-content {
+//     display: block;
+//   }
+//   .card-img {
+//     display: none;
+//   }
+// }
+
+
+#social-good {
```
Comment (alexgleason) [inline www/assets/styles/card_styles.scss]: Why all these ID's? IMO this shouldn't be in CSS

File: www/assets/styles/home_styles.scss [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  a {
+    color: white;
+    &:hover {
+        color: #4A9DA7;
+      }
+  }
+}
+
+
+#social-background {
```
Comment (alexgleason) [inline www/assets/styles/home_styles.scss]: Same thing as above

File: www/assets/styles/style.scss [hunk 5]
```diff
@@ -53,59 +48,115 @@ a.button {
...
+      }
+    }
   }
 }
 
+.external-logo {
+  width: 20px;
+  height: 20px;
+}
+
```
Comment (alexgleason) [inline www/assets/styles/style.scss]: Lots of excess whitespace here

File: www/index.njk [hunk 6]
```diff
@@ -0,0 +1,61 @@
+{% extends "_base.njk" %}
+
+{% block extra_styles %}
+<link rel="stylesheet" href="/assets/styles/home_styles.css">
```
Comment (alexgleason) [inline www/index.njk]: I prefer to indent block contents. I think it looks nicer and it's consistent with how we do for loops. I noticed you indented the block below this but you don't do it consistently.

File: www/index.njk [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+        <div class="starfield"></div>
+        <div class="twinkle"></div>
+        <div class="clouds"></div>
+      </div>
+      <nav>
+        <div class="nav">
+          <ul>
+            <li><a href="/services" class="button">Services</a></li>
+            <li><a href="/about" class="button">About</a></li>
+            {# <li><a href="/portfolio" class="button">Portfolio</a></li> #}
```
Comment (alexgleason) [inline www/index.njk]: When you comment code please add another comment above explaining why it's commented

File: server.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+const harp = require('wick')
+const app = express()
+
+const PORT = process.env.PORT || 3000
+
+app.use(harp.mount(__dirname + '/www'))
+app.use('/node_modules', express.static(__dirname + '/node_modules'))
+
+console.log(`Server is running at http://localhost:${PORT}`)
+app.listen(PORT)
```
Comment (alexgleason) [inline server.js]: Do we even need this whole file? I don't think it's being called by `npm run start`. It's unnecessary.

Comment (marykatefain) [reply server.js]: I'm not sure, I thought it may be necessary for nunjucks?

File: www/assets/styles/card_styles.scss (hunk 2 above)
Comment (marykatefain) [reply www/assets/styles/card_styles.scss]: How about I just add a comment? My goal is to fix it in like 4 days haha

File: www/assets/styles/style.scss (hunk 5 above)
Comment (marykatefain) [reply www/assets/styles/style.scss]: Yeeeeeahhhh this was mostly just to separate the background drama from the rest of the css. This stuff is not to be touched haha

**Pull request merged**